source/main/index.js

```diff
@@ -202,14 +203,29 @@ const onAppReady = async () => {
     await safeExit();
   });
 
-  buildAppMenus(mainWindow, cardanoNode, locale, { isUpdateAvailable: false });
+  buildAppMenus(mainWindow, cardanoNode, locale, {
+    isUpdateAvailable: false,
+    isNavigationEnabled: false,
+  });
+
+  await enableApplicationMenuNavigationChannel.onReceive(
+    () =>
+      new Promise((resolve) => {
+        buildAppMenus(mainWindow, cardanoNode, locale, {
+          isUpdateAvailable: false,
+          isNavigationEnabled: true,
+        });
+        resolve();
+      })
+  );
 
   await rebuildApplicationMenu.onReceive(
     (data) =>
       new Promise((resolve) => {
         locale = getLocale(network);
         buildAppMenus(mainWindow, cardanoNode, locale, {
           isUpdateAvailable: data.isUpdateAvailable,
+          isNavigationEnabled: true,
```

Comment:
@yakovkaravelov this should not be always set to true

Reply:
@nikolaglumac This is fixed. Thanks.